### DISCUSSION TITLE
Whisper Auto Captions for Final Cut Pro

### DISCUSSION THREAD

**Discussion by shaishaicookie** [ggml-org/whisper.cpp#912] [Show and tell]
Hey everyone, I've developed a macOS app that automatically generates subtitles for Final Cut Pro.

Special thanks to ggerganov, whisper.cpp dramatically improved the transcription speed of the whisper model on Macs without GPUs.

You can check out a demo of this app on YouTube (https://youtu.be/2_QOACyuZIk)
and download it from our website (https://whisperautocaptions.com/).

The source code is also available on GitHub (https://github.com/shaishaicookie/fcpx-auto-captions).

**Comment by chenzilu1990** [comment]
good job i like it
